src/components/panes/CameraTest/CameraTest.tsx

```diff
@@ -21,15 +21,11 @@ const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     mainContainer: {
       display: 'block',
-      [theme.breakpoints.only('md')]: {
-        marginLeft: '3em',
-        width: '70%',
-      },
     },
     paper: {
       padding: '2em',
       borderRadius: '8px',
-      width: '388px',
+      width: '368px',
```

Comment:
Now that I think about it, it doesn't feel right to have a fixed width on the camera test. This component should be flexible and just use the width of the grid component that it's in. 

But, that can make the camera test too big on mobile, so we can prevent that with maxWidth. I found that something around `max-width: '450px'` looks nice.